### ISSUE TITLE
Dark mode links have poor contrast

### ISSUE THREAD

**Issue by kenjenkins** [pomerium/documentation#1000]
Page: n/a

## What's incorrect or missing

In dark mode, the contrast between the purple link color and the background is low.

Chrome DevTools shows body links at a contrast ratio of 2.97:
![Screen Shot 2023-09-20 at 11 49 40 AM](https://github.com/pomerium/documentation/assets/51246568/21704754-654f-4a86-80af-219a2927d906)

The breadcrumb showing the current page has a lighter background, and DevTools calculates the contrast ratio here as 2.61:
![Screen Shot 2023-09-20 at 11 49 16 AM](https://github.com/pomerium/documentation/assets/51246568/439df711-e2e7-41ff-b870-31142c76b35b)

## What's the resolution?

Can we lighten the link color in dark mode, to increase the contrast?

`#b499ff` should meet the WCAG AAA contrast ratio recommendation for the body text links:

![Screen Shot 2023-09-20 at 11 57 51 AM](https://github.com/pomerium/documentation/assets/51246568/9cb4463f-3a45-4aaf-bf7e-f413046a697a)
